internal/venice-client-common/src/main/java/com/linkedin/venice/utils/CollectionUtils.java

```diff
@@ -166,6 +168,51 @@ public Set<Entry<K, V>> entrySet() {
     }));
   }
 
+  public static void populateWithPassThroughConfigs(VeniceProperties props, ConfigSetter configSetter) {
```

Comment:
IMHO this doesn't belong here. Let's move it back to abstract class `DataWriterComputeJob`

Reply:
It is not doing anything exclusive to `DataWriterComputeJob`

Reply:
a. It appears to be used exclusively by classes that extend DataWriterComputeJob.
b. Is CollectionUtils the right place for this method? The class seems to focus on generic collection-related utilities, and this usage looks fairly specific.

Reply:
a) yes, "used exclusively", but not "doing anything exclusively" which should determine the placement.
b) any collection can use it to, you can refer the newly added tests.
but for sake of faster review, i will move it to DataWriterComputeJob